### DISCUSSION TITLE
Netlify Identity no longer recommended

### DISCUSSION THREAD

**Discussion by J2thearo** [sveltia/sveltia-cms#284] [General]
Netlify Support has informed me that Netlify Identity is no longer recommend and will not be supported at this time. I just wanted to let you know about this because you planned to integrate Identity better.

**Comment by kyoshino** [comment]
Thanks for letting me know! I can’t find any public announcement in their docs, forums, etc., but the [Netlify Identity widget](https://github.com/netlify/netlify-identity-widget) has been abandoned for a couple of years, so it’s probably time to move on. Now I need to develop both a Netlify Identity widget alternative and a Git Gateway alternative to recreate the Netlify CMS ecosystem 🤣

**Reply by kyoshino** [reply]
[Updated the README](https://github.com/sveltia/sveltia-cms/commit/d440247d2f392a546d58c2f0144b9bc64c3aec15) to reflect the current situation.

**Comment by kyoshino** [comment]
Their official announcement came today: https://www.netlify.com/changelog/deprecation-netlify-identity/
